tests/server/sockfilt.c

```diff
@@ -589,6 +589,7 @@ static DWORD WINAPI select_ws_wait_thread(LPVOID lpParameter)
           }
         }
         /* there is some data available, stop waiting */
+        logmsg("[select_ws_wait_thread] data available on DISK: %d", handle);
```

Comment:
handle is technically a pointer

Reply:
thanks, resolved via force-pushed new version